components/Button/style.module.scss

```diff
@@ -24,16 +25,28 @@
   &--default {
     background: $color-snow;
     box-shadow: 0 1px 1px 0 rgba(0, 0, 0, 0.15);
+
+    &:hover {
```

Comment:
I know we don't have guidelines established for this but we should consider having our focus styles match our hover styles in addition to the blue outline.
```suggestion
    &:hover,
    &:focus {
```

Reply:
No guidelines, no problem. Good catch. *Commits suggestion*